benchmarks/burstgpt_loadgen/convert.py

```
@@ -0,0 +1,294 @@
+    )
+    return parser.parse_args()
+
+
+def load_csv(filepath):
+    """Load CSV file into a pandas DataFrame."""
+    try:
+        df = pd.read_csv(filepath)
+        print(f"Successfully loaded {filepath}")
+        print(f"DataFrame shape: {df.shape}")
+        print(f"Columns: {list(df.columns)}")
+        print(f"First few rows:")
```

Ruff (0.13.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)